akka-http2-support/src/main/scala/akka/http/impl/engine/http2/framing/Http2FrameParsing.scala

```diff
@@ -25,7 +24,7 @@ private[http] object Http2FrameParsing {
       if (payload.hasRemaining) {
         val id = payload.readShortBE()
         val value = payload.readIntBE()
-        if (isKnownId(id)) readSettings(Setting(SettingIdentifier.byId(id), value) :: read)
+        if (SettingIdentifier.isKnownId(id)) readSettings(Setting(SettingIdentifier.byId(id), value) :: read)
         else readSettings(read)
```

Comment:
I guess we should DEBUG log when we ignore settings. Can be added in this PR or in a follow-up.

Reply:
I can try to do a small follow-up PR for logging in the following days, but I'd rather see this going alone so it makes it for the next 10.1 patch release!

Reply:
Sounds good, thanks @bjaglin.